auction-app-backend/src/main/java/com/example/auctionapp/response/PaymentResponse.java

```diff
@@ -0,0 +1,37 @@
+package com.example.auctionapp.response;
+
+public class PaymentResponse {
+    private boolean success;
```

Comment:
If payment fails, aren't we going to throw an exception.
Do we need this field, also error message is unnecessary